### PR TITLE
test: fixed compilation errors in helm

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: core/test/integ/src/plugins/kubernetes/helm/common.ts [hunk 1]
```diff
@@ -103,29 +104,16 @@ describe("Helm common functions", () => {
...
-      expect(await containsSource(module)).to.be.false
-    })
-  })
-
   describe("renderTemplates", () => {
     it("should render and return the manifests for a local template", async () => {
-      const module = graph.getModule("api")
-      const imageModule = graph.getModule("api-image")
+      const deployAction = graph.getDeploy("api")
+      const buildImageAction = graph.getBuild("api-image")
```
Comment (Orzelius) [inline core/test/integ/src/plugins/kubernetes/helm/common.ts:110]: just wondering why is this one a build?

File: core/test/integ/src/plugins/kubernetes/helm/common.ts [hunk 2]
```diff
@@ -611,10 +594,10 @@ ${expectedIngressOutput}
     })
 
     it("localMode should always take precedence over devMode when add a --set flag", async () => {
-      const module = graph.getModule("api")
-      module.spec.valueFiles = []
-      const gardenValuesPath = getGardenValuesPath(module.buildPath)
-      expect(await getValueArgs(module, true, true)).to.eql([
+      const action = await garden.resolveAction({ action: graph.getDeploy("api"), log, graph })
+      action.getSpec().valueFiles = []
```
Comment (Orzelius) [inline core/test/integ/src/plugins/kubernetes/helm/common.ts:598]: I don't think this line does anything. If you want to modify the spec you need to go into the `._config.spec` itself. You can get a typed version with the trick I messaged you in slack

edit: also happens in a couple places other than here as well

**Review by Orzelius — APPROVED** [review]
you can fix the things I mentioned in the comment if you want to I guess

File: core/test/integ/src/plugins/kubernetes/helm/common.ts (hunk 1 above)
Comment (vvagaytsev) [reply core/test/integ/src/plugins/kubernetes/helm/common.ts:110]: The module `api-image` is used for docker build only. there are no services defined in it. So, the build option looked like the best match.

File: core/test/integ/src/plugins/kubernetes/helm/common.ts (hunk 2 above)
Comment (vvagaytsev) [reply core/test/integ/src/plugins/kubernetes/helm/common.ts:598]: Yes, it does. The method `getSpec()` is a shortcut for `._config.spec`, and it returns a reference to a mutable object. So we can easily modify the state of the object over the reference.

Comment (vvagaytsev) [reply core/test/integ/src/plugins/kubernetes/helm/common.ts:599]: Added some explicit typing in https://github.com/garden-io/garden/pull/3155/commits/f9dd825477a1e39c36ca15f665645a86006aa2d2

**Pull request merged**